packages/node-cli/src/core/authentication/AuthenticationService.js

```diff
@@ -16,7 +16,8 @@ const DEFAULT_AUTH_ID_PROPERTY = 'defaultAuthId';
 let CACHED_DEFAULT_AUTH_ID;
 module.exports = class AuthenticationService {
 	constructor(executionPath) {
-		assert(executionPath);
+		assert(executionPath)
+		CACHED_DEFAULT_AUTH_ID = null;
```

Comment:
I think moving CACHED_DEFAULT_AUTH_ID as a property of the class would be better. Each instance of the class would have it's own cached default auth id